### PR TITLE
Changelog for v2.183.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

**Review by MichaelSquires — APPROVED** [review]

File: synapse/tools/changelog.py [hunk 1]
```diff
@@ -222,13 +222,25 @@ def _compareForms(self, curv, oldv, outp: s_output.OutPut) -> dict:
                     deprecated_props_noiface[prop] = cpinfo
                     continue
 
+                okeys = set(opinfo.keys())
+                nkeys = set(cpinfo.keys())
+
+                if nkeys - okeys:
+                    # We've removed a key from the prop def.
```
Comment (mikemoritz) [inline synapse/tools/changelog.py:229]:
```suggestion
                    # We've added a key to the prop def.
```

File: synapse/tools/changelog.py [hunk 2]
```diff
@@ -317,13 +329,25 @@ def _compareIfaces(self, curv, oldv, outp: s_output.OutPut) -> dict:
                     deprecated_props[prop] = cpinfo
                     continue
 
+                okeys = set(opinfo.keys())
+                nkeys = set(cpinfo.keys())
+
+                if nkeys - okeys:
+                    # We've removed a key from the prop def.
```
Comment (mikemoritz) [inline synapse/tools/changelog.py:336]:
```suggestion
                    # We've added a key to the prop def.
```

**Review by MichaelSquires — DISMISSED** [review]

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,41 @@
 Synapse Changelog
 *****************
 
+v2.182.0 - 2024-10-09
```
Comment (Cisphyx) [inline CHANGELOG.rst:9]:
```suggestion
v2.183.0 - 2024-10-09
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,41 @@
...
+=====================
+
+Model Changes
+-------------
+- Fix an issue where the ``:path:base``, ``:path:dir``, and ``:path:ext``
+  secondary properties were marked readonly on the ``it:fs:file``,
+  ``it:exec:file:add``, ``it:exec:file:del``, ``it:exec:file:read``, and
+  ``it:exec:file:write`` forms.
+  (`#3942 <https://github.com/vertexproject/synapse/pull/3942>`_)
+- See :ref:`userguide_model_v2_182_0` for more detailed model changes.
```
Comment (Cisphyx) [inline CHANGELOG.rst:19]:
```suggestion
- See :ref:`userguide_model_v2_183_0` for more detailed model changes.
```

File: docs/synapse/userguides/model_updates/update_v2_182_0.rst [hunk 5]
```diff
@@ -0,0 +1,78 @@
+
+
+.. _userguide_model_v2_182_0:
```
Comment (Cisphyx) [inline docs/synapse/userguides/model_updates/update_v2_182_0.rst:3]: Need to update this file to 183

**Review by Cisphyx — APPROVED** [review]

**Review by mikemoritz — APPROVED** [review]

**Pull request merged**